tests/e2e/serial/main_test.go

```diff
@@ -1933,6 +1934,169 @@ func TestSuspendScanSettingDoesNotCreateScan(t *testing.T) {
 	}
 }
 
+func TestConfigureNetworkPolicy(t *testing.T) {
+	f := framework.Global
+	suiteName := "test-configure-network-policy"
+	suiteNameNoPass := "test-configure-network-policy-no-pass"
+	variableName := "ocp4-var-network-policies-namespaces-exempt-regex"
+	// Create a dummy namespace to test the network policy
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test-configure-network-policy",
+		},
+	}
+	err := f.Client.Create(context.TODO(), ns, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), ns)
+
+	err = f.AssertVariableExists(variableName, f.OperatorNamespace)
+	if err != nil {
+		t.Skip("Content doesn't have variable '%s' required for testing", variableName)
+		return
+	}
+
+	nsList := corev1.NamespaceList{}
+	err = f.Client.List(context.TODO(), &nsList)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	regextValue := ""
+
+	for _, ns := range nsList.Items {
+		if strings.HasPrefix(ns.Name, "openshift-") || strings.HasPrefix(ns.Name, "kube-") {
+			continue
+		}
+		regextValue = regextValue + ns.Name + "|"
+	}
+
+	regextValue = regextValue + ns.ObjectMeta.Name
+
+	tp := &compv1alpha1.TailoredProfile{
```

Comment:
nit: Similar naming comment as above with the suites (could do something like `compliantTailoredProfile` or `tpCompliant`.

Then on line 2010, something like `nonCompliantTailoredProfile`, or `tpNonCompliant`.